src/css/index.scss

```diff
@@ -1,6 +1,7 @@
 @import "variables.scss";
 @import "../components/QuestionsList/QuestionsList.scss";
 @import "../components/QuestionDetail/QuestionDetail.scss";
+@import "../components/QuestionCreate/QuestionCreate.scss";
```

Comment:
J'ai rien dit sur l'utilisation de sass lors du Hackathon, mais je pense qu'on pourrait utiliser `styled-components`.  C'est une solution de CSS-in-JS qui marche très bien avec React, qui est compatible avec TS et qui nous évite d'avoir le style à 4 km du component